kfs-ld/src/main/resources/edu/arizona/kfs/module/ld/spring-ld.xml

```diff
@@ -21,5 +21,103 @@
                 <value>classpath:org/kuali/kfs/module/ld/document/datadictionary/overrides/*.xml</value>
             </list>
         </property>
+        <property name="triggerNames">

```

Comment:
There aren't actually any triggers defined in spring-ld, so this is an unneeded override.